pkg/streamingpromql/testdata/ours/limit.test

```diff
@@ -0,0 +1,245 @@
+load 5m
+  http_requests{job="api-server", instance="0", group="production"}	0+10x10
+  http_requests{job="api-server", instance="1", group="production"}	0+20x10
+  http_requests{job="api-server", instance="0", group="canary"}		0+30x10
+  http_requests{job="api-server", instance="1", group="canary"}		0+40x10
+  http_requests{job="api-server", instance="2", group="canary"}		0+50x10
+  http_requests{job="api-server", instance="3", group="canary"}		0+60x10
+  http_requests{job="api-server", instance="histogram_1", group="canary"}       {{schema:0 sum:10 count:10}}x11
+  http_requests{job="api-server", instance="histogram_2", group="canary"}       {{schema:0 sum:20 count:20}}x11
+  mixed{instance="1"}       {{schema:0 sum:20 count:20}}x2 0+10x5 {{schema:0 sum:20 count:20}}x2 0+10x2
+  mixed{instance="2"}       0+10x5 {{schema:0 sum:20 count:20}}x2 0+10x2 {{schema:0 sum:20 count:20}}x2
+  missing{instance="1"}       0+10x5 _ _ _ _ _ 0+10x2 _
+  missing{instance="2"}       _ _ _ _ _ 0+10x5 _ 0+10x2 _
+  nans{instance="1"}        NaN -NaN 0+10x5 NaN -NaN
+  nans{instance="2"}        0+10x5 NaN -NaN NaN -NaN
```

Comment:
The use of both `_` and `...x...` points makes it difficult to quickly understand the test data - could you please expand the sample data to make it clearer what values are at each point?

```suggestion
  missing{instance="1"}       10 20 30 40 50 _ _ _ _ _ 10 20 _
  missing{instance="2"}       _ _ _ _ _ 10 20 30 40 50 _ 10 20 _
  nans{instance="1"}        NaN -NaN 10 20 30 40 50 NaN -NaN
  nans{instance="2"}        10 20 30 40 50 NaN -NaN NaN -NaN
```